packages/edit-site/src/components/dataviews/view-grid.js

```diff
@@ -1,3 +1,71 @@
-export function ViewGrid() {
-	return 'Grid';
+/**
+ * WordPress dependencies
+ */
+import {
+	__experimentalGrid as Grid,
+	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
+	Card,
+	CardBody,
+	CardMedia,
+	FlexBlock,
+	Placeholder,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import FieldActions from './field-actions';
+
+export function ViewGrid( { data, fields, view, actions } ) {
+	const mediaField = fields.find(
+		( field ) => field.id === view.layoutConfig.mediaField
+	);
+	const visibleFields = fields.filter(
+		( field ) =>
+			! view.hiddenFields.includes( field.id ) &&
+			field.id !== view.layoutConfig.mediaField
+	);
+	return (
+		<Grid gap={ 6 } columns={ 2 }>
+			{ data.map( ( item, index ) => {
+				return (
+					<Card key={ index }>
+						<CardMedia>
+							{ ( mediaField &&
+								mediaField.render( { item } ) ) || (
+								<Placeholder
+									withIllustration
+									style={ {
+										width: '100%',
+										minHeight: '200px',
+									} }
+								/>
+							) }
+						</CardMedia>
+
+						<CardBody>
+							<HStack justify="space-between" alignment="top">
+								<FlexBlock>
+									<VStack>
+										{ visibleFields.map( ( field ) => (
```

Comment:
The visible fields are just rendered one after the other under the "media" and on the left of the "actions menu".